scripts/createAndroidKeys.ts

```diff
@@ -0,0 +1,71 @@
+import childProcess from 'child_process'
+import { randomBytes } from 'crypto'
+// import fs from 'fs'
+import { join } from 'path'
+import prompts from 'prompts'
+
+let _currentPath = __dirname
+
+const main = async (): Promise<void> => {
+  // let alias = ''
+
+  mylog(_currentPath)
+  const newpath = join(_currentPath, '..', 'keystores')
+  mylog(newpath)
+  chdir(newpath)
+  const password = (await randomBytes(16)).toString('hex')
+
+  const { alias } = await prompts({
+    name: 'alias',
+    type: 'text',
+    message: 'Enter app alias (ie. "edge-develop")',
+    validate: (v: string) => v.trim() !== ''
+  })
+
+  const { encryptionKey } = await prompts({
+    name: 'encryptionKey',
+    type: 'text',
+    message: 'Enter encryptionKey provided by Google Play under the option "Export and upload a key from Java keystore"',
+    validate: (v: string) => v.trim() !== ''
+  })
+
+  call(
+    `keytool -genkey -keystore ${alias}-keystore.jks -keyalg RSA -keysize 2048 -validity 10000 -alias ${alias} -dname "cn=Unknown, ou=Unknown, o=Unknown, c=Unknown" -storepass ${password} -keypass ${password}`
+  )
+  mylog(`Enter the following password at the prompts: ${password}`)
+  call(`java -jar ~/bin/pepk.jar --keystore=${alias}-keystore.jks --alias=${alias} --output=${alias}.zip --include-cert --encryptionkey=${encryptionKey}`)
+
+  // Uncomment out if creating a separate upload key
+
+  // call(
+  //   `keytool -genkey -keystore ${alias}-upload-keystore.jks -keyalg RSA -keysize 2048 -validity 10000 -alias ${alias}-upload -dname "cn=Unknown, ou=Unknown, o=Unknown, c=Unknown" -storepass ${password} -keypass ${password}`
+  // )
+  // call(
+  //   `keytool -export -rfc -keystore ${alias}-upload-keystore.jks -alias ${alias}-upload -file ${alias}-upload-cert.pem -storepass ${password} -keypass ${password}`
+  // )
+
+  mylog('Keystore created in keystores directory. Save the following keystore password. It cannot be recovered if lost.')
+  mylog('********************************')
+  mylog(password)
+  mylog('********************************')
+}
+
+const mylog = console.log
```

Comment:
I'd argue to move this constant to the top of the file

Reply:
Per prior comments from William, utility function should be at the end of the file so you can see the primary routines at the top.

Reply:
Ah, fair.